scripts/cron.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------------
+# START CONFIGURATION SECTION
+# ----------------------------------------------------------------------
+
+# name of the dabase to load
+# this script assumes the user running it has access to the database
+DATABASE=${DATABASE:-"bety"}
+
+# hostname
+FQDN=${FQDN:-"$(hostname -f)"}
+
+# psql options
+# this allows you to add any other options
+PG_OPT=${PG_OPT:-""}
+
+# ID's used in database, see load.bety.sh
+# These ID's need to be unique for the sharing to work. If you want
+MYSITE=${MYSITE:-""}
+
+# location where to write the results, this will be a tar file
+OUTPUT=${OUTPUT:-"$PWD/dump"}
+
+# ----------------------------------------------------------------------
+# END CONFIGURATION SECTION
+# ----------------------------------------------------------------------
+
+SCRIPTS_DIR="$( dirname $0 )"
+
+# parse command line options
+while getopts d:f:hm:o:p: opt; do
+  case $opt in
+  d)
+    DATABASE="$OPTARG"
+    ;;
+  f)
+    FQDN="$OPTARG"
+    ;;
+  h)
+    echo "$0 [d database] [-f fqdn] [-h] [-m my siteid] [-o folder] [-p psql options]"
+    echo " -d database, default is bety"
+    echo " -f hostname, default is '$(hostname -f)'"
+    echo " -h this help page"
+    echo " -m site id, default is not set and will be guessed based on hostname"
+    echo " -o output folder where dumped data is written, default is dump"
+    echo " -p additional psql command line options, default is empty"
+    exit 0
+    ;;
+  m)
+    MYSITE="$OPTARG"
+    ;;
+  o)
+    OUTPUT="$OPTARG"
+    ;;
+  p)
+    PG_OPT="$OPTARG"
+    ;;
+  esac
+done
+
+# guess site
+if [ "${MYSITE}" == "" ]; then
+    MYSITE=$(sudo -u postgres psql ${PG_OPT} -d ${DATABASE} -t -c "SELECT sync_host_id FROM machines WHERE hostname='${FQDN}';")
+fi
+MYSITE=${MYSITE//[[:space:]]/}
+if [ "${MYSITE}" == "" ]; then
+    MYSITE="99"
+fi
+
+MY_START_ID=$(sudo -u postgres psql ${PG_OPT} -d ${DATABASE} -t -c "SELECT sync_start FROM machines WHERE sync_host_id=${MYSITE};")
+MY_START_ID="${MY_START_ID//[[:space:]]/}"
+MY_LAST_ID=$(sudo -u postgres psql -d bety -t -c "SELECT sync_end FROM machines WHERE sync_host_id=${MYSITE};")
+MY_LAST_ID="${MY_LAST_ID//[[:space:]]/}"
+
+if [ "${MY_START_ID}" == "" -o "${MY_LAST_ID}" == "" ]; then
+    # this value should be constant, do not change
+    ID_RANGE=1000000000
+    MY_START_ID=$(( MYSITE * ID_RANGE + 1 ))
+    MY_LAST_ID=$(( MY_START_ID + ID_RANGE - 1 ))
+fi
+
+# export variables for other scripts
+export DATABASE FQDN MYSITE MY_START_ID MY_LAST_ID PG_OPT
+
+# setup thredds catalog
+"${SCRIPTS_DIR}/thredds.sh" > /home/carya/thredds/catalog.xml
```

Comment:
no that should become I guess an option to pass in. I will create an issue for this and fix in the future.